File: ios/精通iOS开发/6 view switcher/6 view switcher/YellowViewController.m
//
//  YellowViewController.m
//  6 view switcher
//
//  Created by andyron on 16/1/29.
//  Copyright © 2016年 andyron. All rights reserved.
//

#import "YellowViewController.h"

@implementation YellowViewController

- (IBAction)yellowButtonPressed {
    UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Yellow View Button Pressed" message:@"You pressed the button on the blue view" delegate:nil cancelButtonTitle:@"Yep, I did." otherButtonTitles:nil];
    [alert show];
}
@end

File: iOS/old/Beginning iPhone Development with Swift 2, 2nd Edition/6/View Switcher/View Switcher/YellowViewController.swift
//
//  YellowViewController.swift
//  View Switcher
//
//  Created by andyron on 16/3/1.
//  Copyright © 2016年 andyron. All rights reserved.
//

import UIKit

class YellowViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */
    @IBAction func yellowButtonPressed(sender: UIButton) {
        let alert = UIAlertController(title: "Yellow View Button Pressed",
        message: "You pressed the button on the yellow view",
        preferredStyle: .Alert)
        let action = UIAlertAction(title: "Yep, I did", style: .Default,
        handler: nil)
        alert.addAction(action)
        presentViewController(alert, animated: true, completion: nil)
    }
}
